.github/workflows/test.yml

```diff
@@ -0,0 +1,34 @@
+name: Test
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+permissions:
+  contents: read
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version:
+          - 20
+          - 18
+          - 16
+          - 14
+          - 12
+          - 10
+          - 8
```

Comment:
should we drop these versions?

Reply:
I will do it in separated PR with dependencies update